externs/shaka/player.js

```diff
@@ -707,6 +708,9 @@ shaka.extern.AdvancedDrmConfiguration;
  * @property {!Object.<string, string>} clearKeys
  *   <i>Forces the use of the Clear Key CDM.</i>
  *   A map of key IDs (hex) to keys (hex).
+ * @property {boolean} clearKeysAsRaw
+ *   <i>Defaults to false.</i> <br>
+ *   True to send the clearkeys without processing.
```

Comment:
I didn't understand what `clearKeysAsRaw` meant.  Even this explanation didn't clarify it for me.  When I read the DrmEngine changes, I finally understood.

I think it would be much clearer as something like `drm.clearKeysInBase64`, with an explanation that otherwise, they will be treated as hex (the default).  You can also update the description of the `clearKeys` object above.

An alternative would be to use string length to automatically detect base64 vs hex encoding, since we know that both key ID and key should be 128 bits.

Reply:
If it seems better to you, I do it automatically, for me it is much simpler.